### PR TITLE
{lib}[foss/2022a] PyWBGT v1.0.0 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/p/PyWBGT/PyWBGT-1.0.0-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+description = """Cython source code for estimating wet bulb globe temperature (WBGT) from datasets of standard
+meterological measurements using models developed by Liljegren et al (2008)"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {'openmp': True}
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('SciPy-bundle', '2022.05'),
+    ('xarray', '2022.9.0'),
```
Comment (boegel) [inline easybuild/easyconfigs/p/PyWBGT/PyWBGT-1.0.0-foss-2022a.eb:16]:
```suggestion
    ('xarray', '2022.6.0'),

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3165.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/21c01fd3a1cfc207668b70e98bea37b0 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node304.hydra.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/smoors/de1272520d34339d5d57619d660a2580 for a full test report.

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @boegel!

**Pull request merged**